go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/microsoft/terraform-provider-power-platform
 
-go 1.21
+go 1.21.9
```

Comment:
why don't we move to 1.22 directly? newest release to this day is `1.22.3`

https://go.dev/doc/devel/release

Reply:
Meke sense, I will update the version.